packages/commonwealth/test/devnet/cosmos/tokenBalanceFetching.spec.ts

```diff
@@ -12,13 +12,59 @@ import BN from 'bn.js';
 import { use as chaiUse, expect } from 'chai';
 import chaiAsPromised from 'chai-as-promised';
 import { RedisCache } from 'common-common/src/redisCache';
+import { ChainNodeAttributes } from 'server/models/chain_node';
+import {
+  BalanceType,
+  ChainBase,
+  ChainNetwork,
+  ChainType,
+} from '../../../../common-common/src/types';
 import models from '../../../server/database';
 import { BalanceSourceType } from '../../../server/util/requirementsModule/requirementsTypes';
 import { TokenBalanceCache } from '../../../server/util/tokenBalanceCache/tokenBalanceCache';
 import { delay } from '../../util/delayUtils';
 
 chaiUse(chaiAsPromised);
 
+const setUpDatabase = (): Promise<void> => {
+  return new Promise(async (resolve) => {
+    try {
+      await models.sequelize.sync({ force: true });
```

Comment:
Can this test work without this? Devnet tests already require `yarn reset-db && yarn migrate-db` so resetting here should be avoided if possible as it is a costly step.

Reply:
I was thinking these probably don't belong in the devnet tests. It made some sense to co-locate them with the cosmos native cache tests, but they don't use the devnet, so maybe they should be in integration anyway?

But yes, I will address this either way.

Reply:
What do you mean they don't use the devnet tests? All tests in `/cosmos/tokenBalanceFetching.spect.ts` including the cache tests use the devnet

Reply:
I just meant these CW721 tests, because they test the actual stargaze chain. I will keep them here.

I guess ideally, we would enable cosmwasm on a devnet to test this, but the complexity of minting NFTs etc seems overkill when we can just ping stargaze.

Reply:
>Can this test work without this .sync reset

removed

Reply:
Ahh I see. I think long-term we will want to enable cosmwasm anyway so that we can run complex tests on-chain like we do for EVM. What do you think would be the point estimate for setting up cosmwasm on a devnet and then deploying a CW721 contract?

Reply:
3 to 5.